app/src/androidTest/java/org/mozilla/tv/firefox/helpers/SessionLoadedIdlingResource.kt

```diff
@@ -25,9 +26,20 @@ class SessionLoadedIdlingResource : IdlingResource {
 
         val session = sessionManager.selectedSession
 
+        /**
+         * Used for [org.mozilla.tv.firefox.ui.YouTubeNavigationTest].
+         * This will cause the test to wait until a specific condition is true before moving on.
```

Comment:
EDIT: it looks like this is all deleted in a later commit.  Try to clean up previous commits when that happens (or at worst, go back and add a `TODO: deleted in a later commit`), otherwise this wastes a fair amount of review time.  (this is a mistake I've made frequently too, it's easy to forget)

~This kdoc would probably be better placed on the declaration on line 17.  Also, slight nit on the formatting:~

```kotlin
/**
 * This will cause the test to wait until a specific condition is true before moving on.
 *
 * @sample [org.mozilla.tv.firefox.ui.YouTubeNavigationTest]
 */
```
~(it's pretty generic functionality, it seems likely that we'll use it elsewhere too)~

Reply:
Ah, sorry. I left the commit just in case I needed to go back to it again. But there are better ways I can handle that.

Reply:
All good, as I said, I've done the same many times.